### PR TITLE
TRW-915 - converting all margin, padding, and font stuff to rems

### PR DESCRIPTION
**Pull request by troyblank**
There is some places margin is centering things that I did not convert but most of it's all rems now. I made the base 10px and the body `font-size: 1.4rem` so that all default text will not be so tiny. At some point we can bust our paddings out into some vars, but this is a step in making that easier at least.

I wanted to do this before moving forward with universal search as it would have been a pain in building it with rems.

### CODE REVIEW

File: assets/sass/base/silents/_containers.scss [hunk 1]
```diff
@@ -67,7 +49,7 @@
         padding: 8px 10px;
         float: left;
         border-right: 1px solid $color-grey;
-        line-height: rem(17px);
+        line-height: 1.6rem;
```
Comment (tVeratti) [inline assets/sass/base/silents/_containers.scss]: Is it okay for these to be off 1px after the change?

File: assets/sass/demo/partials/_colors.scss [hunk 2]
```diff
@@ -2,20 +2,20 @@
...
+        margin-bottom: 2rem;
     }
 
     li {
         width: 100%;
         height: 32px;
         border-bottom: 1px solid $color-black;
-        font-size: rem(14px);
+        font-size: 1em;
         list-style: none;
```
Comment (tVeratti) [inline assets/sass/demo/partials/_colors.scss]: `em` here

**Comment by tVeratti** [comment]
Is this giving the desired effect without setting `html { font-size: 62.5%; }`?

File: assets/sass/base/vars/_typography.scss [hunk 3]
```diff
@@ -1 +1 @@
-$base-font-size: 14px;
+$base-font-size: 10px;
```
Comment (troyblank) [inline assets/sass/base/vars/_typography.scss]: @tVeratti so this should be the same as `{ font-size: 62.5%; }`.  Although I have never got why people use that, it assumes browsers will always keep `16px` as the default font size. I don't like to assume and thus I am trying to take matters into our own hands with the `10px`. That is unless there is something magic about that percentage that I am not understand.

File: assets/sass/base/silents/_containers.scss (hunk 1 above)
Comment (troyblank) [reply assets/sass/base/silents/_containers.scss]: Nope it's not, ooop. Pushed up a fix.

File: assets/sass/demo/partials/_colors.scss (hunk 2 above)
Comment (troyblank) [reply assets/sass/demo/partials/_colors.scss]: Good catch, pushed up a change.

File: assets/sass/base/silents/_forms.scss [hunk 4]
```diff
@@ -127,7 +127,7 @@
     left: 0;
     transition: top $animation-time-crisp $animation-curve-crisp, font-size $animation-time-crisp $animation-curve-crisp, color $animation-time-crisp $animation-curve-crisp;
     color: $color-grey-dark;
-    font-size: rem(14px);
+    font-size: 1.6rem;
```
Comment (tVeratti) [inline assets/sass/base/silents/_forms.scss]: `1.4rem`

File: assets/sass/base/silents/_lists.scss [hunk 5]
```diff
@@ -14,31 +14,31 @@
...
     background: $color-grey-lighter;
 
     button {
         @extend %naked-button;
         flex-basis: 0;
         flex-grow: 1;
-        padding: 12px 20px 12px 10px;
-        font-size: rem(14px);
+        padding: 1.2rem 2rem 1.2rem 0;
+        font-size: 1.2rem;
```
Comment (tVeratti) [inline assets/sass/base/silents/_lists.scss]: 1.4rem

File: assets/sass/base/silents/_typography.scss [hunk 6]
```diff
@@ -5,8 +5,8 @@
 }
 
 %header-tertiary {
-    margin: 0 0 15px;
-    font-size: rem(14px);
+    margin: 0 0 1.5rem;
+    font-size: 1.6rem;
```
Comment (tVeratti) [inline assets/sass/base/silents/_typography.scss]: 1.4rem

File: assets/sass/base/vars/_typography.scss (hunk 3 above)
Comment (tVeratti) [reply assets/sass/base/vars/_typography.scss]: That was always the implementation I'd seen passed around. I'm fine with the 10px if it accomplishes the same thing.

File: assets/sass/demo/partials/_colors.scss [hunk 7]
```diff
@@ -2,20 +2,20 @@
...
-        margin-bottom: 20px;
+        margin-bottom: 2rem;
     }
 
     li {
         width: 100%;
         height: 32px;
         border-bottom: 1px solid $color-black;
-        font-size: rem(14px);
+        font-size: 1rem;
```
Comment (tVeratti) [inline assets/sass/demo/partials/_colors.scss]: 1.4rem

File: assets/sass/demo/partials/_senchaPage.scss [hunk 8]
```diff
@@ -28,15 +30,15 @@
...
+    padding: 0 1rem;
     background-color: $color-orange;
     color: $color-white;
 
     li {
         height: 39px;
         padding: 13px 22px 0;
         float: left;
-        font-size: rem(13px);
+        font-size: 1.6rem;
```
Comment (tVeratti) [inline assets/sass/demo/partials/_senchaPage.scss]: 1.3

**Pull request merged**